creepy/subprocess/pypen.py

```diff
@@ -80,12 +82,27 @@ def __init__(self, args, hash: Optional[str] = None, serializable: bool = False,
         try:
             self._send, self._recv = secure_alice(send, recv, make_cipher=self._save_and_make_cipher)
         except Exception:
-            self.detach()
+            self._detach()
+        self._bind_interface()
 
     @property
-    def pid(self):
+    def _pid(self):
```

Comment:
It's not good to make it backward incompatible.

Reply:
What if child has `pid` route?

Reply:
We can add rules to the `App.route`. One of them will be "don't use `pid` name".

Reply:
Child having `pid` route isn't a reason to break backward compatibility. In fact Pypen mimics Popen class, so breaking such backward compatibility shouldn't be an option even to consider.
Secondly, if child has `pid` route, is there anything really bad? Child's pid will just be called instead of Pypen's pid. This issue can be handled in several ways:
For example, you can define also 
```python
_pid = pid
_wait = wait
…
```
Also, you can make this functionality not to be available by default, but turned by passing special param in constructor. It's good to have such an option for backward compatibility anyway.